### PR TITLE
Add TERMINOLOGY.md

### PR DESCRIPTION
**Pull request by dreamerlin**
A basic reference as Chinese-English translation terminology, which can be updated for other repos

### CODE REVIEW

File: TERMINOLOGY.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+| checkpoint | 模型权重文件 |
+| backbone | 主干网络 |
+| cls_head | 分类头 |
+| pipeline | 流水线 |
+| hook | 钩子 |
+| finetune | 微调 |
+| register | 注册器 |
+| tensor | 张量 |
+| benchmark | 基准测试 |
+| recognizer | 动作识别器 |
```
Comment (zhouzaida) [inline TERMINOLOGY.md:18]: 识别器？

File: TERMINOLOGY.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+| detector | 时空动作检测器 |
```
Comment (zhouzaida) [inline TERMINOLOGY.md:19]: 检测器？

File: TERMINOLOGY.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+| localizer | 时序动作检测器 |
```
Comment (zhouzaida) [inline TERMINOLOGY.md:20]: 定位器？

**Review by zhouzaida — APPROVED** [review]
LGTM

File: TERMINOLOGY.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+| register | 注册器 |
+| tensor | 张量 |
+| benchmark | 基准测试 |
+| recognizer | 识别器 |
+| detector | 检测器 |
+| localizer | 定位器 |
+| annotation | 标注 |
+| training schedule | 训练策略 |
+| scheduler | 调度器 |
+| schedule | 调整 |
```
[the remark this answers is not in the document]
Comment (dreamerlin) [reply TERMINOLOGY.md:24]: training schedule -> 训练策略

**Comment by hellock** [comment]
License: 许可证

**Comment by ZwwWayne** [comment]
This PR will be kept updated till almost all OpenMMLab repos release their Chinese documentation.

**Pull request merged**